sdk/js/src/api/http.js

```diff
@@ -15,32 +15,55 @@
 import axios from 'axios'
 import { cloneDeep, get, isObject } from 'lodash'
 
-import { URI_PREFIX_STACK_COMPONENT_MAP, STACK_COMPONENTS_MAP } from '../util/constants'
+import Token from '../util/token'
 import EventHandler from '../util/events'
+import {
+  URI_PREFIX_STACK_COMPONENT_MAP,
+  STACK_COMPONENTS_MAP,
+  AUTHENTICATION_MODES,
+} from '../util/constants'
 
 import stream from './stream/stream-node'
 
 /**
  * Http Class is a connector for the API that uses the HTTP bridge to connect.
  */
 class Http {
-  constructor(token, stackConfig, axiosConfig = {}) {
+  constructor(authentication, stackConfig, axiosConfig = {}) {
```

Comment:
```suggestion
  constructor(authorization, stackConfig, axiosConfig = {}) {
```

imo, it is authorization iformation